staff/daniel-toledo/gamma-template/style.sass

```diff
@@ -0,0 +1,419 @@
+*
+    margin: 0
+    padding: 0
+    box-sizing: border-box
+
+%green
+    color: rgb(28,170,152)
+
+.header
+    background-color: rgb(27,34,42)
+    display: flex
+    flex-direction: column
+    justify-content: center
+
+    &__menu
+        background-color: rgb(248, 248, 248)
+        display: flex
+        flex-direction: row
+        justify-content: space-between
+        align-items: center
+
+    &__logo
+        padding: 10px
+        @extend %green
+
+    &__nav
+        display: none
+    
+    &__burguer
+        border: lightgrey 1px solid
+        border-radius: 0.25em
+        padding: 10px
+        margin-right: 30px
+    
+    &__page-title
+        background-color: rgb(28,170,152)
+
+    &__page-title h1
+        padding: 10px
+        margin-left: 20px
+        color: white
+        font-size: 40px
+        font-family: Arial, Helvetica, sans-serif
+    
+    &__title
+        margin: 1em
+        text-align: center
+        color: white
+    
+    &__title--green
+        @extend %green
+    
+    &__text
+        margin: 1em
+        text-align: center
+        color: white
+        font-family: Arial, Helvetica, sans-serif
+
+    &__images
+        display: flex
+        flex-direction: column
+        justify-content: center
+
+    &__img
+        margin: 1em auto
+        width: 80%
+        max-width: 300px
+    
+    &__picture
+        display: none
+
+.content
+    margin: 4em auto
+    padding: 0px 10px
+
+    &__img
+        width: 100%
+        border: solid lightgrey 1px
+
+    &__title
+        margin: 1em auto
+    
+    &__text
+        margin: 1em auto
+        font-family: Arial, Helvetica, sans-serif
+
+    &__btn
+        cursor: pointer
+        outline: 0
+        padding: 5px 10px
+        background-color: white
+        border: solid lightgrey 1px
+        border-radius: 0.3em
+     
+    &__btn:hover
+         color: rgb(28,170,152)
+
+.content--contact
+    margin-left: 20px
+    margin-right: 20px
+    paddin-top: 0
```

Comment:
Are you sure this paddin-top is working? Check the spelling!

Reply:
Owww yeah!! Good eyes :astonished:

Reply:
changed ;)